section6Arrays/mergeArrays/main.go

```diff
@@ -0,0 +1,27 @@
+package merge_arrays
```

Comment:
pls rename as suggested in other PR 
Also rename the file from main to something more meaningful.
Introduce an interface here. Lets discuss, when you have time.

Reply:
Ok